Gopkg.toml

```diff
@@ -90,8 +90,9 @@
   name = "github.com/juju/naturalsort"
 
 [[constraint]]
-  revision = "0ffd1641ec10b98d99eabb23729a2156244da119"
+  revision = "313a6d1c94a82e242621e4ca42f53de2af3cccf2"
   name = "github.com/juju/os"
+  source = "github.com/SimonRichardson/os"
```

Comment:
Is this line needed?

Reply:
yeah, because my PR hasn't merged, once it has, I can remove this and use the merged commit ref.

Reply:
The PR was merged and I removed this.